Client/Ecosia/MobileMeasurementPartner/SingularSessionParametersMapper.swift

```diff
@@ -0,0 +1,61 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+enum SingularSessionParametersMapper {
+    
+    private struct SingularSessionInfo: Encodable {
```

Comment:
Understood your point with the mapper 👍 

I have to say I still find it a bit strange and that it would be more clear for the Singular networking code to handle it since it is directly related to what the API expects, even if this means we have to change Core too when the API changes. I also wonder since our `ios-core` repo is private and this one is public, if we don't want to "hide" the APIs logic on purpose.

Even so, feel free to leave it like this, it is not a big issue and we can re-iterate on it after we evolve the networking layer 🙂

Reply:
Agreed on re-iterate on it later